manifests/a/Alibaba/DingTalk/7.8.5-Release.250710001/Alibaba.DingTalk.locale.zh-CN.yaml

```
@@ -0,0 +1,41 @@
+# Created with komac v2.12.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.10.0.schema.json
+
+PackageIdentifier: Alibaba.DingTalk
+PackageVersion: 7.6.10-Release.82010807

```

```suggestion
PackageVersion: 7.8.5-Release.250710001
```

Komac should've updated this 🤔